.golangci.yaml

```diff
@@ -38,6 +41,8 @@ linters-settings:
   gomoddirectives:
     # List of allowed `replace` directives. Default is empty.
     replace-allow-list:
-      # FIXME: google/ko requires 0.22, remove when knative will work with 0.22+
-      - k8s.io/apimachinery
+      # FIXME: remove this pin when knative/eventing#6451 is merged
+      - knative.dev/eventing
+      # FIXME: remove this pin when knative-sandbox/reconciler-test#294 is merged
+      - knative.dev/reconciler-test
```

Comment:
Remove those exclude, after merge of https://github.com/knative-sandbox/reconciler-test/pull/294

Reply:
Fixed in b85f141be77dee08942c478fc6c19f85d885d5a3